Pepper_bringup and navigation

fmrico opened the pull request:
Compilation for:

- Pepper_bringup (includes rgbd perception)
- Navigation

esteve commented on build_ros1.sh:
```diff
@@ -34,6 +34,7 @@ docker run -it --rm \
   -e PYTHON2_MAJOR_VERSION=${PYTHON2_MAJOR_VERSION} \
   -e PYTHON2_MINOR_VERSION=${PYTHON2_MINOR_VERSION} \
   -e PYTHON3_VERSION=${PYTHON3_VERSION} \
+	-e INSTALL_ROOT=${INSTALL_ROOT} \
```
There's a mix of tabs and spaces in the changes, could you change it to always use spaces?

fmrico commented:
Fixed in 51982c5

Atom tabs were not correctly configured ;)

I also changed the INSTALL_DIR to not be hidden, as comment is Readme.md suggested

esteve commented:
@fmrico thanks! I don't know why the comment about `.ros-root` got added to `README.md`, though I prefer to keep that change separate in another pull request. I can revert the change to keep it as `.ros-root` and push it or if you prefer to do it yourself, I'm ok either way.

fmrico commented:
No problem, I changed it back to .ros-root (hidden) in dde4d91

Thanks!!!

esteve commented:
@fmrico I've merged #5 , can you rebase this to fix the conflicts? Thanks!

fmrico commented:
Incorporated Tom's changes in ceee17a

esteve commented:
@fmrico thanks for rebasing the changes. Let me know if you need help addressing the other comments.

fmrico commented:
@esteve I think I have resolved all the issues related to your comments: fixed tabs, ros-root is hidden, and all consistent with Tom's work. All compiled and tested in Pepper. Do you think I have to resolved anything else to apply this PR?

esteve commented:
@fmrico there's still a few comments in https://github.com/esteve/ros2_pepper/pull/4#pullrequestreview-94078693, and a comment about duplicated code in `build_ros1_dependencies.sh` that is now hidden because of the rebase

esteve commented:
@fmrico I've set the keyserver to the hkps pool (https://github.com/esteve/ros2_pepper/commit/4bdd40285ef5c2ce3d3fb291097f2642b46771a4) and added the paths to the compiler tools as environment variables in the docker files (https://github.com/esteve/ros2_pepper/commit/7a33187f3590f3621cbe13a42c3557a45f4eea18) so there's no need to set them for each crosscompiled project. The latter should simplify the scripts a bit.
